### PR TITLE
Fixing image title for examples

### PR DESCRIPTION
**Pull request by matthewmccarthy11**
Fixing the location of images from /content/assets to /assets.

### CODE REVIEW

**Comment by matthewmccarthy11** [comment]
Here is some more context @rufuspollock 

Complication: The images won't pop up in the preview. Links are set relative path to file and the images show up in obsidian fine. Four different image urls we're experimented with: 

-/assets/imgname
-/content/assets/imgname
- assets/imgname
- content/assets imgname

Another image was attempted to be upload to the 'why' page to see if it would work, and this did not work either. 

Hypothesis: It is not something to do with the 'examples' page, and I don't know know why they are not showing up.

**Comment by rufuspollock** [comment]
@matthewmccarthy11 that's a great analysis. The issue was in fact simple and unrelated to what you were doing ...

I did a check and could see that this url was not working ...

https://d1739655.developmentalspaces-org.pages.dev/assets/agaia.jpg

It should be ... so i went and checked and there was a mess up in symlinking the `content/assets` folder into the `public` folder. This is Flowershow detail you need not worry about ...

The key thing for you was that that url above was the thing that should be working ... and if it is not something is clearly wrong ...

File: content/why.md [hunk 1]
```diff
@@ -27,4 +27,6 @@ Build the personal and collective capacities for this, which we believe include
 
 #### Why more people with capacities?
 
-Address the metacrisis, build a flourishing future for humanity -- or handle the inevitable crisis with greater resilience etc.
\ No newline at end of file
+Address the metacrisis, build a flourishing future for humanity -- or handle the inevitable crisis with greater resilience etc.
+
+![[assets/Bildung.jpg]]
```
Comment (rufuspollock) [inline content/why.md:32]: I don't want an image here ...

File: ..content/assets/agaia.jpg.md
Comment (rufuspollock) [inline ..content/assets/agaia.jpg.md:1]: this seems a weird rename ... i don't think this file should be here ...???

File: content/agaia.md [hunk 2]
```diff
@@ -12,11 +12,11 @@ contributors: Matthew McCarthy
 
 Agaia is an organization that provides coliving retreats for impact-driven entrepreneurs, nomads, and communities in Italy.
 
-![[content/assets/agaia.jpg]]
+![[assets/agaia.jpg]]
```
Comment (rufuspollock) [inline content/agaia.md:15]: all of these will work once we merge the fix i put in main ...

**Review by rufuspollock — CHANGES_REQUESTED** [review]

**Pull request merged**